Duplicati/Server/Runner.cs

```diff
@@ -764,7 +789,10 @@ private static void UpdateMetadata(Duplicati.Server.Serialization.Interface.IBac
                 backup.Metadata["LastBackupDuration"] = r.Duration.ToString();
 
                 if (r.CompactResults != null)
-                    UpdateLastCompactMetadata(backup, r.CompactResults);
+                    UpdateMetadataLastCompact(backup, r.CompactResults);
+
+                if (r.VacuumResults != null)
+                    UpdateMetadataLastVacuum(backup, r.VacuumResults);

```

Comment:
Is it possible for `r.CompactResults.VacuumResults` to be non-null here?

As an aside, I would like to eventually try and change how we deal with all the different results.  Instead of results containing other results, perhaps a list or tree of results can be processed in a more streamlined manner.

Reply:
Nope - it will always be null.  When a backup triggers a compact operation, compact will never then trigger a vacuum.  I dug into that because at first I thought there was the possibility of a double vacuum under normal operation, but there is not.

An automatic compact operation after backup triggers `CompactHandler.DoCompact` which does not call any sort of vacuum operation.

An intentional compact operation triggers `CompactHandler.Run`, which in turn calls `CompactHandler.DoCompact` and then triggers a vacuum.

Took me a little while to wrap my head around all this.